src/Copy/Keys.elm

```diff
@@ -79,8 +79,63 @@ type Key
     | SplitterAffirmation
     | ToGetHelpFromDefinitionLink
     | ToHelpSelfFromDefinitionLink
-
-
-
--- Get Help page
--- Help Self page
+      -- Help Self page
+    | HelpSelfTitle
+    | HelpSelfCategoryLink1
+    | HelpSelfCategoryLink2
+    | HelpSelfCategoryLink3
+    | HelpSelfCategoryLink4
+    | HelpSelfCategoryLink5
+    | ToDefinitionFromHelpSelfLink
+    | ToNotAloneFromHelpSelfLink
+    | ToGetHelpFromHelpSelfLink
+    | ToSelfHelpFromSingleCategoryLink
+      -- Help Self Single Category1
+    | HelpSelfCategoryTitle1
+    | HelpSelfCategoryResource1Title1
```

Comment:
I think if I put the trailing int after Category, this would be more clear

Reply:
I think a bit clearer yes.

Reply:
Good plan. I will move the int.

Reply:
Done.